### PR TITLE
March 8 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | b063e8c |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-417.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240308_amplify.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+### On-premise agents
+
+#### Axway API Management Gateway
+
+The following enhancements and bug fixes are available in this release.
+
+##### Enhancements for Axway API Management Agent
+
+* **Clean variable settings** - Support for enable the settomg of the INPUTS_CLEAN_REMOVED and INPUTS_CLOSE_REMOVED variables to TRUE or FALSE has been added.  In addtion, support has been added for the INPUTS_IGNORE_OLDER (default: 1h),
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240308_amplify.md:68]: fix typos

File: content/en/docs/amplify_relnotes/20240308_amplify.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+INPUTS_CLOSE_INACTIVE (default: 10m), and INPUTS_CLEAN_INACTIVE (default: 2h) variables.
+
+#### Bug fixes for Axway API Management Gateway
+
+* **Invoke Policy** - Fixed an issue with a NULL pointer exception when discovering an API with Invoke Policy.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240308_amplify.md:73]: with invoke policy as the inbound security profile.

File: content/en/docs/amplify_relnotes/20240308_amplify.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+There are no known limitations in this release.
+
+#### Apigeed Edge Gateway Agent
+
+The following enhancements are available in this release.
+
+##### Enhancements for Apigee Edge Gateway Agent
+
+* **Discovery from a local file** - The Apigee Edge discovery agent has added support to read API Specifications from the local file system.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240308_amplify.md:85]: can we add few more lines to explain hhow the api specs are discovered? based on tags, etc

File: content/en/docs/amplify_relnotes/20240308_amplify.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+* **Discovery of APIs with no specification - The Apigee Edge Discovery Agent can be configure to discovery APIs with no associated API specification file.  The variable to enable this is APIGEE_SPECCONFIG_UNSTRUCTURED and it must be set to TRUE.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240308_amplify.md:86]: In order to enable it, the variable APIGEE_SPECCONFIG_UNSTRUCTURED must be set to TRUE.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by georgiana-caras — APPROVED** [review]

**Pull request merged**